mapswipe_workers/mapswipe_workers/generate_stats/project_stats.py

```diff
@@ -31,10 +33,18 @@ def write_sql_to_csv(filename: str, sql_query: sql.SQL):
     Use the copy statement to write data from postgres to a csv file.
     """
 
+    temp_file = "temp.csv"
```

Comment:
@laurentS I'm pretty sure that there is a better solution here. How can we avoid using a temp file here?

Reply:
I quickly tested using pandas, but couldn't get to a solution easily. That's why I keep the existing logic, but now use a proper tempory file instead.